### ISSUE TITLE
Adding a 'Beginner Friendly CLI" version of the Getting Started Guide

### ISSUE THREAD

**Issue by skylarposler** [beetbox/beets#5259]
### Discussed in https://github.com/beetbox/beets/discussions/5258

<div type='discussions-op-text'>

<sup>Originally posted by **skylarposler** May 24, 2024</sup>
Creating a ‘Beginner Friendly CLI Getting Started Guide’ with more in-depth step by step instructions and screenshots to help eliminate errors upon installation and configuration, will help more inexperienced CLI users navigate and set up beets correctly. </div>

**Comment by khrystianc** [comment]
If no one has started or done this, I would like to work on a product for this issue resolution.

**Comment by skylarposler** [comment]
Hi! I'm currently working on the windows version of this. If you would like, the MacOS could benefit from this as well.

**Comment by khrystianc** [comment]
Thank you for getting back to me so quickly. This issue is all yours. I am working under windows as well.

**Comment by RollingStar** [comment]
Thank you! I just gave it a very light skim but being accessible to non-command-line users should help our reach tremendously.

I can't find where I suggested it, but I think "editable" (running from python source .py files) should be our default/preferred/only way of running beets in the documentation.

https://beets.readthedocs.io/en/stable/faq.html#run-the-latest-source-version-of-beets

It encourages new users to become new developers of light features/bug fixes, and encourages understanding how beets works.

**Comment by justinmayer** [comment]
I am not sure that “editable” Python installation should be presented in the documentation as the default/preferred way of running Beets. But I am _definitely_ sure that it shouldn’t be presented as the **only** way.

For folks who _want_ to contribute to Beets, I think it's great to encourage them to run the latest version in editable mode. Many people do not fall in that category, and those users are arguably better served by packaged, stable releases.

Plus, [that section of the documentation](https://beets.readthedocs.io/en/stable/faq.html#run-the-latest-source-version-of-beets) needs work before it could even be used to encourage folks to run in editable mode. For starters, `setup.py` does not exist any longer, so the suggested `python setup.py install` option will fail.

Perhaps most importantly, it is not clear from those docs how to install Beets in editable mode **and** install optional packages. For example, how would someone install Beets in editable mode with the following optional packages? This section of the documentation does not cover this at all.

    python -m pip install beets[chroma,discogs,fetchart,lastgenre,lyrics,mpdstats,plexupdate,web]

Editable mode is awesome, and I use it all the time for projects that I maintain. But it is often not the best installation option for end users, and frequently becomes a crutch for projects that often go way, *way* too long between packaged releases. (*[Ahem](https://github.com/beetbox/beets/compare/v1.6.0...v2.0.0).*)

**Comment by antebudimir** [comment]
Hi! 

Has there been any progress on this one?

**Comment by justinmayer** [comment]
@antebudimir: Please allow me to refer you to [Any Updates?](https://justinmayer.com/posts/any-updates/)

**Comment by antebudimir** [comment]
Yeah, that does and doesn't make sense because it happened not once that I stumbled upon a repo that didn't follow this logic and there actually was something happening in the meantime that wasn't put to paper. 😉

Thanks either way.
